unpackaged/config/ps_testing/classes/PS_TestPaymentService.cls

```diff
@@ -36,13 +36,28 @@
 */
 public with sharing class PS_TestPaymentService {
 
-    private static final String BASE_PURCHASE_CALL_BODY = '{"amount":25,"email":"test@test.test","firstName":"John","lastName":"Doe","metadata":{},';
-    private static final String TOKEN_FIELD_NAME = 'token';
-    private static final String STATUS_FIELD_NAME = 'status';
+    private static final String BASE_PURCHASE_CALL_BODY = '{"amount":2500,"email":"test@test.test","firstName":"John","lastName":"Doe","metadata":{},';
```

Comment:
The amount must be multiplied with the currency denominator in order for Elevate to process a credit card charge (the $.25 cannot be processed).